docker-compose.yaml

```diff
@@ -22,7 +22,7 @@ services:
       dockerfile: Dockerfile
     env_file:
       - .env
-    restart: always
+    restart: "no"
```

Comment:
Changed because if the tests fail with `always`, the container will continually restart until the docker-compose command is killed.

Reply:
It can't be. `no` is a yaml variable which evaluates to false, which we don't want.